source/install/troubleshooting.rst

```diff
@@ -19,9 +19,40 @@ General Troubleshooting
 -----------------------
 Some of these suggestions can be done directly, and others may need consultation from your network administrator.
 
-- Take a look at the logs (``mattermost.log`` and NGINX logs) for errors.
-- You can also search the error messages online - existing solutions can often be applied.
-- Open **System Console > General > Logging** and set File Log Level to **DEBUG**. Make sure to revert to **INFO** after troubleshooting to save disk space.
+Take a Look at the Logs
+~~~~~~~~~~~~~~~~~~~~~~~
+
+This article covers accessing the numerous logs created by Mattermost applications.
+
+**Mattermost Server**
+
+1. In the *System Console* go to *Environment > Logging* and note down the path to the *File Log Directory*
+2. The server log file is called *mattermost.log* and can be opened with a standard text editor or shared directly.
```

Comment:
```suggestion
2. The server log file is called ``mattermost.log`` and can be opened with a text editor or shared directly.
```
How does one get to the server log file, and what type of sharing would be done (and how?).

Reply:
that is explained under `more`. the path can be set individually. you can share it by copy paste in a mail for example.

Reply:
The numbering of this section isn't rendering correctly - would you be open to committing the suggestion and reviewing the numbering? It's showing as a single line in the file view.

Reply:
<img width="951" alt="Screen Shot 2020-04-25 at 11 23 00" src="https://user-images.githubusercontent.com/4223530/80276203-4e73ca00-86e7-11ea-8f65-b6f485418a62.png">